<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDialogClass</class>
 <widget class="QDialog" name="UDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UrbGet - Get What You Want, Man</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Download</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Page:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;CurrentFolder&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Manga:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>35</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;CurrentFile&gt;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>35</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Manga</string>
   </property>
   <widget class="QLineEdit" name="MangaEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>95</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OnePiece</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Url of the first page:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>75</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of the Manga:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumberEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>95</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UrlEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>http://img55.onemanga.com/mangas/00000004/000100772/01.jpg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>75</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Mangas:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>471</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Output Settings</string>
   </property>
   <widget class="QLineEdit" name="DirEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination Directory:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>35</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Warning: it is not checked if dir is valid!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
